config/config.go

```diff
@@ -1,27 +1,35 @@
 package config
 
-import "github.com/ian-kent/gofigure"
+import (
+	"time"
 
-//Config contains configurable details for running the service
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config contains configurable details for running the service
 type Config struct {
-	BindAddr string `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"The port to bind to"`
+	BindAddr        string        `envconfig:"BIND_ADDR"`
+	HierarchyAPIURL string        `envconfig:"HIERARCHY_API_URL"`
+	DbAddr          string        `envconfig:"HIERARCHY_DATABASE_ADDRESS"`
+	ShutdownTimeout time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
+	CodelistAPIURL  string        `envconfig:"CODE_LIST_URL"`
 }
 
 var configuration *Config
 
 // Get configures the application and returns the configuration
 func Get() (*Config, error) {
-	if configuration != nil {
-		return configuration, nil
-	}
-
-	configuration = &Config{
-		BindAddr: ":22600",
+	if configuration == nil {
+		configuration = &Config{
+			BindAddr:        ":22600",
+			HierarchyAPIURL: "http://localhost:22600",
+			DbAddr:          "bolt://localhost:7687",
+			ShutdownTimeout: time.Duration(5 * time.Second),
```

Comment:
It has the same effect, but there is no need for time.Duration() as time.Second is already a duration. `ShutdownTimeout: 5 * time.Second,`